packages/rollup-plugin/test/rollup-js-exports.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import { rollupRunner } from './test-kit/rollup-runner';
+import { getProjectPath } from './test-kit/test-helpers';
+
+describe('StylableRollupPlugin - js exports', function () {
+    this.timeout(30000);
+
+    const project = 'js-exports';
+
+    const runner = rollupRunner({
+        projectPath: getProjectPath(project),
+        entry: './src/index.js',
+        pluginOptions: {
+            stcConfig: './project/stylable.config.js',
```

Comment:
why do we need `stcConfig` here? I think we  can remove it and delete the file

Reply:
no need - removed